docs/website/docs/getting-started/tflite.md

```diff
@@ -0,0 +1,199 @@
+# TFLite Integration
+
+IREE supports coming and running TensorFlow-Lite programs stored as
+[TFLite flatbuffers](https://www.tensorflow.org/lite/guide). These files can be
+imported into an IREE compatible format then compiled to a series of backends.
+
+## Prerequisites
+
+Install TensorFlow-Lite specific dependencies using pip:
+
+```shell
+python -m pip install \
+  iree-compiler \
+  iree-runtime \
+  iree-tools-tflite
+```
+
+# TFLite via Command Line
```

Comment:
am I seeing double, or is this "TFLite via Command Line" section duplicated? :P

Reply:
Done. VSCode's github integration has been acting funny, definitely fixed this before sending it out -_-.